.ci_support/build_all.py

```diff
@@ -79,6 +80,10 @@ def build_all(recipes_dir, arch):
                         version = tuple([int(x) for x in version.split('.')])
                         sdk_version = max(sdk_version, deployment_version, version)
 
+            if 'channel_sources' in text:
+                config = load(text, Loader=BaseLoader)
+                channel_urls += config['channel_sources'][0].split(',')
```

Comment:
I think what we want is to error if one recipe has `channel_sources` and it is not equal in all the recipes. Appending sources really doesn't make sense to me.

Reply:
@isuruf Would it be possible that only one recipe needs custom channel_sources and not the others?